### ISSUE TITLE
Add a way to set a tooltip delay for specific node locally

### ISSUE THREAD

**Issue by filantus** [godotengine/godot-proposals#4738]
### Describe the project you are working on

UI improvement.

### Describe the problem or limitation you are having in your project

Built-in function `Object _make_custom_tooltip(for_text: String) virtual const` it's a great thing.

But it's becomes useless in case if required to show tooltip instantly (or with non default delay) in some UI element (only).

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Described above.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

For example add some property or/and some methods in node space like:
```gdscript
tooltip_delay
set_tooltip_delay(value: float)
get_tooltip_delay() -> float
```

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Workaround - set tooltip delay globally, but it's not allow to have different tooltip delays in different ui elements.

And actually even global tooltip delay change don't work right now. See: https://github.com/godotengine/godot/issues/9030

### Is there a reason why this should be core and not an add-on in the asset library?

Core UI improvement.

**Comment by Calinou** [comment]
If you want to display a tooltip instantly, I think it's better to use a custom tooltip system for this. The built-in tooltip system is intended to remain basic, and it mainly targets non-game applications.

A custom tooltip system also allows you to have transitions, formatting, and more.

**Comment by filantus** [comment]
@Calinou Yep, thanks for advice. Actually i'm already did it (custom system). Though it took some time to overcome through some troubles.
But i'm also thinking it may be useful to have ability to set local tooltip delay (for built-in tooltips). Even for simple tooltips. And in non game apps too.
